kernel/src/process/rlimit.rs

```diff
@@ -1,37 +1,73 @@
 // SPDX-License-Identifier: MPL-2.0
-
+// FIXME: The resource limits should be respected by the corresponding subsystems of the kernel.
 #![allow(non_camel_case_types)]
```

Comment:
Need to insert a blank line between the FIXME and the `allow` attribute. It looks like the FIXM targets the `allow` attribute.

Reply:
Get it.